src/snowflake/cli/plugins/nativeapp/commands.py

```diff
@@ -229,6 +230,11 @@ def app_teardown(
 @app.command("deploy", requires_connection=True)
 @with_project_definition("native_app")
 def app_deploy(
+    files: Optional[List[Path]] = typer.Argument(
+        default=None,
+        show_default=False,
+        help="Files to stage, relative to deploy root. [default: all local changes]",
```

Comment:
This help text is used directly to generate our documentation. We should make this a bit more clear, something like:
Path of the file, relative to the deploy root, to be uploaded to a stage. Default: sync all local changes with the stage.

Or something similar.